frontend/src/features/company/components/AdminCompanyForm.tsx

```diff
@@ -0,0 +1,28 @@
+import RHFTextField from '@shared/RHF/RHFTextField'
+import RHFAutocomplete from '@shared/RHF/RHFAutocomplete';
+import { Stack } from '@mui/material'
+
+export default function AdminCompanyForm({
+  domains, states
+}: { domains: string[]; states: { code: string; name: string }[] }) {
+  return (
+    <Stack spacing={2}>
+      <RHFTextField name="name" label="Company Name" fullWidth />
```

Comment:
ux polish : 
Autofocus first field
```suggestion
      <RHFTextField name="name" label="Company Name" fullWidth autoFocus />

```